src/main/java/syncsquad/teamsync/controller/MainWindowController.java

```diff
@@ -106,9 +106,9 @@ public MainWindowController(Stage primaryStage, Logic logic) {
 
         // TODO: If we are adding `nextWeek` or `previousWeek` actions
         //  we should create a currentWeekController to handle the logic
-        this.viewModel.getCurrentWeekViewModel().currentWeekProperty()
+        this.logic.getCurrentWeek().currentWeekProperty()
```

Comment:
No changes needed but, you should avoid using logic directly. By right MainWindowController should not have access to logic, only MainWindowViewModel. So the by right way should be:
`MainWindowController` -> `MainWindowVM.currentWeekProperty()` -> `return meetingListVM.currentWeekProperty()`

`MeetingListVM` should take in logic and subscribe to the currentWeekProp from logic